app-libs/stf/src/lib.rs

```diff
@@ -81,11 +81,10 @@ pub enum StfError {
 	StorageHashMismatch,
 	InvalidStorageDiff,
 }
-#[allow(clippy::large_enum_variant)]
```

Comment:
If the clippy lint [large_enum_variant](https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant) should be suppressed let me know and I will drop the patch for it.